scripts/abi_bindings.sh

```diff
@@ -52,33 +55,53 @@ echo "Building Contracts"
 cd $AVALANCHE_INTERCHAIN_TOKEN_TRANSFER_PATH/contracts
 forge build --skip test --force --extra-output-files abi bin
 
+function convertToLower() {
+    if [ "$ARCH" = 'arm64' ]; then
+        echo $1 | perl -ne 'print lc'
+    else
+        echo $1 | sed -e 's/\(.*\)/\L\1/'
+    fi
+}
+
 contract_names=($CONTRACT_LIST)
 
 # If CONTRACT_LIST is empty, use DEFAULT_CONTRACT_LIST
 if [[ -z "${CONTRACT_LIST}" ]]; then
     contract_names=($DEFAULT_CONTRACT_LIST)
 fi
 
+generate_bindings() {
```

Comment:
For consistency, should this be declare a function also? Also I think it would help readability if the functions were all moved to the top of the file prior to any commands getting executed.

Reply:
`ShellCheck`, which I've tried and liked and which I know the avalanche-go has built into CI, says skip `function` everywhere: https://www.shellcheck.net/wiki/SC2112

Reply:
moved the function up to the top without function declaration